examples/example-gateway/clients/quux/quux.go

```diff
@@ -21,16 +21,49 @@
 package quux
 
 import (
+	barClient "github.com/uber/zanzibar/examples/example-gateway/build/clients/bar"
 	module "github.com/uber/zanzibar/examples/example-gateway/build/clients/quux/module"
-	"github.com/uber/zanzibar/runtime"
+	zanzibar "github.com/uber/zanzibar/runtime"
+	"go.uber.org/zap"
+	"golang.org/x/net/context"
 )
 
 // Client is a custom client that does nothing yet
 type Client interface{}
 
-type quux struct{}
+type quux struct {
+	client *zanzibar.TChannelClient
```

Comment:
This client is meant to be a demo for a custom client. It doesn't make sense to fill in a tchannel client which should be defined in thrift.